packages/sync/src/Functions.php

```diff
@@ -1,43 +1,65 @@
 <?php
+/**
+ * Utility functions to generate data synced to wpcom
+ *
+ * @package automattic/jetpack-sync
+ */
 
 namespace Automattic\Jetpack\Sync;
 
 use Automattic\Jetpack\Constants;
-/*
+
+/**
  * Utility functions to generate data synced to wpcom
  */
-
 class Functions {
 	const HTTPS_CHECK_OPTION_PREFIX = 'jetpack_sync_https_history_';
 	const HTTPS_CHECK_HISTORY       = 5;
 
+	/**
+	 * Return array of Jetpack modules.
+	 *
+	 * @return array
```

Comment:
Would be nice if all `return` have their description as well.

Reply:
For the most part they are self-explanatory. My primary goal with this PR is to get the file into enough compliant that it doesn't fail the precommit hooks with every change. I'm not sure how it got in without passing that with `packages` being on the whitelist.